### PR TITLE
chore: App router - Fix errors in settings pages & add remaining pages

### PR DESCRIPTION
**Pull request by hbjORbj**
## What does this PR do?

- Fixes CAL-4309, CAL-4310
- Remove references to /pages for remaining routes: remaining ones are all pages in `settings/teams` and some pages in `settings/organizations`
- Remove unneeded layout files from `future/settings/(settings)/organizations`
- Move pages that do not need Settings layout out of `(settings)` route group
- Move pages that do need Settings layout into `(settings)` route group

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A - I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

Navigate to the following pages for both app/pages router:

- settings/teams/[id]/appearance
- settings/teams/[id]/billing
- settings/teams/[id]/members
- settings/teams/[id]/onboard-members
- settings/teams/[id]/profile
- settings/teams/new

### CODE REVIEW

File: packages/features/ee/teams/pages/team-billing-view.tsx [hunk 1]
```diff
@@ -1,34 +0,0 @@
-import { usePathname } from "next/navigation";
```
Comment (hbjORbj) [inline packages/features/ee/teams/pages/team-billing-view.tsx:1]: this file isn't being used anywhere

File: apps/web/app/future/settings/(settings)/organizations/appearance/layout.tsx [hunk 2]
```diff
@@ -1,5 +0,0 @@
-import { WithLayout } from "app/layoutHOC";
```
Comment (hbjORbj) [inline apps/web/app/future/settings/(settings)/organizations/appearance/layout.tsx:1]: `(settings)` route group already has a layout file, so this layout is an extra layer that is no longer needed. Same for other layout files I deleted.

File: apps/web/app/future/settings/(settings)/organizations/profile/page.tsx [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
 
 export const generateMetadata = async () =>
   await _generateMetadata(
     (t) => t("profile"),
     (t) => t("profile_org_description")
   );
 
+const Page = async () => {
+  const session = await getServerSessionForAppDir();
+  const t = await getFixedT(session?.user.locale || "en");
```
Comment (sean-brydon) [inline apps/web/app/future/settings/(settings)/organizations/profile/page.tsx:15]: I wonder if we should move this to a UTIL to get translations for active user in appdir

**Comment by hbjORbj** [comment]
For both pages router / app router, tested
- [x] settings/teams/[id]/appearance
- [x] settings/teams/[id]/billing
- [x] settings/teams/[id]/members
- [x] settings/teams/[id]/onboard-members
- [x] settings/teams/[id]/profile
- [x] settings/teams/new
- [x] settings/teams

**Comment by hbjORbj** [comment]
- [ ] settings/organizations/[id]/about
- [ ] settings/organizations/[id]/add-teams
- [ ] settings/organizations/[id]/onboard-members
- [ ] settings/organizations/admin-api
- [ ] settings/organizations/appearance
- [ ] settings/organizations/attributes/[id]/edit
- [ ] settings/organizations/attributes/create.tsx
- [ ] settings/organizations/attributes/index.tsx
- [ ] settings/organizations/billing.tsx
- [ ] settings/organizations/dsync.tsx
- [ ] settings/organizations/general.tsx
- [ ] settings/organizations/members.tsx
- [ ] settings/organizations/new/index.tsx
- [ ] settings/organizations/privacy.tsx
- [ ] settings/organizations/profile.tsx
- [ ] settings/organizations/sso.tsx
- [ ] settings/organizations/teams/other/[id]/appearance.tsx
- [ ] settings/organizations/teams/other/[id]/members.tsx
- [ ] settings/organizations/teams/other/[id]/profile.tsx

File: apps/web/app/future/settings/(settings)/organizations/profile/page.tsx (hunk 3 above)
Comment (hbjORbj) [reply apps/web/app/future/settings/(settings)/organizations/profile/page.tsx:15]: Yeah, good point. Can you create a separate PR for that?

File: apps/web/app/future/settings/(admin)/admin/page.tsx [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
-
-import { getLayout } from "@components/auth/layouts/AdminLayoutAppDir";
 
 export const generateMetadata = async () =>
   await _generateMetadata(
     () => "Admin",
     () => "admin_description"
   );
 
-export default WithLayout({ getServerLayout: getLayout, Page: LegacyPage })<"P">;
```
Comment (sean-brydon) [inline apps/web/app/future/settings/(admin)/admin/page.tsx:13]: Are we missing something from here - page/layout seems to be missing in the new changes

File: packages/features/ee/organizations/pages/settings/other-team-members-view.tsx [hunk 5]
```diff
@@ -136,25 +135,27 @@ const MembersView = () => {
...
+                type="button"
+                color="primary"
+                StartIcon="plus"
+                className="ml-auto"
+                onClick={() => setShowMemberInvitationModal(true)}
+                data-testid="new-member-button">
+                {t("add")}
+              </Button>
+            ) : (
+              <></>
```
Comment (sean-brydon) [inline packages/features/ee/organizations/pages/settings/other-team-members-view.tsx:154]: Looks like we are losing the CTA for app dir?

**Review by sean-brydon — CHANGES_REQUESTED** [review]

File: apps/web/app/future/settings/(admin)/admin/page.tsx (hunk 4 above)
Comment (hbjORbj) [reply apps/web/app/future/settings/(admin)/admin/page.tsx:13]: (admin) route group has a layout file, so all the nested pages inside (admin) route group will get passed the layout and page wrapper

File: packages/features/ee/organizations/pages/settings/other-team-members-view.tsx (hunk 5 above)
Comment (hbjORbj) [reply packages/features/ee/organizations/pages/settings/other-team-members-view.tsx:154]: Oh yeah, I wrote a TODO comment; After I add it, we are good to go

**Review by sean-brydon — APPROVED** [review]
Happy with addressing the CTA in a follow up so we can test on staging. Lets create an issue for it so we don't forget

**Pull request merged**